### PR TITLE
Update  MIP84

### PR DESCRIPTION
**Pull request by Gala-G**
Update MIP84
Add templates

### CODE REVIEW

**Review by flexgod76 — CHANGES_REQUESTED** [review]
Can you explain this in a more simplex form

**Review by flexgod76 — APPROVED** [review]

File: MIP84/MIP84.md [hunk 1]
```diff
@@ -45,19 +45,27 @@ Defines the POVE Debt Ceiling calculation.
 **MIP84c6: Modify the Collateralization Ratio of the Protocol-Owned Vault Emulation**
 Defines the ability to modify the POVE Debt Ceiling calculation.
 
-**MIP84c7: Daily Collateral Purchase Rate** Defines the daily rate at which the protocol will purchase POVE collateral.
+**MIP84c7: Dss-kiln parameters**
+Defines the dss-kiln parameters used to do recurring purchases of POVE collateral
 
-**MIP84c8: Modify Daily Collateral Purchase Rate**
-Defines the ability to modify the Daily Collateral Purchase Rate.
+**MIP84c8: Modify the dss-kiln parameters**
```
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
**MIP84c8: Modify the dss-kiln Parameters**
```

File: MIP84/MIP84.md [hunk 2]
```diff
@@ -45,19 +45,27 @@ Defines the POVE Debt Ceiling calculation.
 **MIP84c6: Modify the Collateralization Ratio of the Protocol-Owned Vault Emulation**
 Defines the ability to modify the POVE Debt Ceiling calculation.
 
-**MIP84c7: Daily Collateral Purchase Rate** Defines the daily rate at which the protocol will purchase POVE collateral.
+**MIP84c7: Dss-kiln parameters**
+Defines the dss-kiln parameters used to do recurring purchases of POVE collateral
```
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
Defines the dss-kiln parameters used to do recurring purchases of POVE collateral.
```

File: MIP84/MIP84.md [hunk 3]
```diff
@@ -45,19 +45,27 @@ Defines the POVE Debt Ceiling calculation.
...
-Defines the ability to modify the Daily Collateral Purchase Rate.
+**MIP84c8: Modify the dss-kiln parameters**
+Defines the ability to modify the dss-kiln parameters
 
-**MIP84c9: Protocol Surplus and the Emulated Surplus Buffer Upper Limit** Defines the relationship between the protocol's surplus and the POVE's buffer.
+**MIP84c9: Protocol Surplus and the Emulated Surplus Buffer Upper Limit**
+Defines the relationship between the protocol's surplus and the POVE's buffer.
+
+**MIP84c10: Modify Emulated Surplus Buffer Upper Limit**
+Defines the ability to modify the Emulated Surplus Buffer Upper Limit
```
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
Defines the ability to modify the Emulated Surplus Buffer Upper Limit.
```

File: MIP84/MIP84.md [hunk 4]
```diff
@@ -45,19 +45,27 @@ Defines the POVE Debt Ceiling calculation.
...
 
 **MIP84c10: Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling**
 Defines a virtual maximum for collateral assets in the POVE.
 
 **MIP84c11: Modify Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling**
 Defines the ability to modify the POVEC's Virtual Price Ceiling.
 
+**MIP84c13: Long-term replacement of the Protocol-Owned Vault Emulation**
+Defines how the POVE must eventually be replaced by a smart contract based Protocol Owned Vault
```
Comment (blimpa) [inline MIP84/MIP84.md:67]:
```suggestion
Defines how the POVE must eventually be replaced by a smart contract-based Protocol-Owned Vault.
```

File: MIP84/MIP84.md (hunk 3 above)
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
Defines the ability to modify the Emulated Surplus Buffer Upper Limit.
```

File: MIP84/MIP84.md [hunk 5]
```diff
@@ -45,19 +45,27 @@ Defines the POVE Debt Ceiling calculation.
...
+
 ## Motivation
 
 The Protocol-Owned Vault is a key element of the Endgame Plan that helps achieve real resilience and decentralization by increasing the amount of Dai backed by Staked ETH. The Protocol-Owned Vault Emulation is an approach to emulate the outcome of operating a Protocol-Owned Vault, but only using existing features of the Maker Protocol as it exists today. This is achieved by using the ability to set a negative lower limit to the surplus buffer in order to emulate a vault with an available debt ceiling, and using the Pause Proxy and dss-kiln to accumulate collateral assets.
```
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
The Protocol-Owned Vault is a key element of the Endgame Plan that helps achieve real resilience and decentralization by increasing the amount of Dai backed by Staked ETH. The Protocol-Owned Vault Emulation is an approach to emulate the outcome of operating a Protocol-Owned Vault, but only using existing features of the Maker Protocol as it exists today. This is achieved by using the ability to set a negative lower limit to the Surplus Buffer in order to emulate a vault with an available debt ceiling, and using the Pause Proxy and dss-kiln to accumulate collateral assets.
```

File: MIP84/MIP84.md [hunk 6]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
 
 The POVE will operate based on manual processes defined in this MIP that are controlled directly by MKR holders using Executive Votes. The Risk Core Unit is mandated with following these pre-defined processes, making parameter adjustment proposals, and is given the emergency powers to de-risk POVE activity if it becomes necessary, including the immediate cessation of additional collateral purchases. This can be done through weekly governance polls, or if necessary be bundled directly into weekly or emergency Executive Votes.
 
-The POVE is exclusively controlled and owned by Maker Governance, and no third-party, individual, multisig or other point of control has any technical access to the POVE or its assets in any way.
+The actions that can be proposed by Risk Facilitators to derisk the POVE in a weekly governance poll, or directly into an emergency or weekly executive vote include, but are not limited to:
```
Comment (blimpa) [inline MIP84/MIP84.md:91]: I'd rephrase this or break it down into independent clauses, like:
- "RISK-001 Facilitators can propose actions to derisk the POVE via weekly governance polls or into emergency or weekly executive votes. These actions include but are not limited to:"

File: MIP84/MIP84.md [hunk 7]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
+* Stopping the dss-kiln from purchasing  more collateral
```
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
* Stopping the dss-kiln from purchasing  more collateral.
```

File: MIP84/MIP84.md [hunk 8]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
+* Installing a new dss-kiln with custom parameters to sell collateral for Dai to wind down the debt of the POVE
```
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
* Installing a new dss-kiln with custom parameters to sell collateral for Dai to wind down the debt of the POVE.
```

File: MIP84/MIP84.md [hunk 9]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
+* Partially or fully liquidate the vault using a custom method
```
Comment (blimpa) [inline MIP84/MIP84.md:94]:
```suggestion
* Partially or fully liquidating the vault using a custom method.
```

File: MIP84/MIP84.md [hunk 10]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
+* Modifying any of the parameters that are relevant for derisking the POVE. Parameters can only be modified this way if it makes them more conservative and reduces risk, it cannot be used to increase risk, such as by lowering the Collateralization Ratio parameter.
```
Comment (blimpa) [inline MIP84/MIP84.md:95]:
```suggestion
* Modifying any of the parameters that are relevant for derisking the POVE. Parameters can only be modified this way if it makes them more conservative and reduces risk. This action cannot be used to increase risk, such as by lowering the Collateralization Ratio parameter.
```

File: MIP84/MIP84.md [hunk 11]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
+- **Minimum Frozen Period**: 1 week.
+- **Governance Cycle**: Monthly.
+
+Facilitators with relevant mandates may propose weekly polls to amend the Collateralization Ratio of the Protocol-Owned Vault Emulation when they deem necessary.
 
-Any Dai available in the Pause Proxy, whether from POVEDC or Starting Collateral, is used to accumulate Lido Staked ETH, and later EtherDai, using a dss-kiln implementation sourcing liquidity from a highly liquid DEX.
+MIP84c6 subproposals must use the template located at [MIP84c6-Subproposal-Template.md](https://github.com/makerdao/mips/blob/master/MIP84/MIP84c6-Subproposal-Template.md).
 
-The rate of accumulation is up to 500,000 Dai per day, if Dai is available.
+### MIP84c7: Dss-kiln parameters
```
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
### MIP84c7: dss-kiln parameters
```

File: MIP84/MIP84.md [hunk 12]
```diff
@@ -45,19 +45,27 @@ Defines the POVE Debt Ceiling calculation.
 **MIP84c6: Modify the Collateralization Ratio of the Protocol-Owned Vault Emulation**
 Defines the ability to modify the POVE Debt Ceiling calculation.
 
-**MIP84c7: Daily Collateral Purchase Rate** Defines the daily rate at which the protocol will purchase POVE collateral.
+**MIP84c7: Dss-kiln parameters**
```
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
**MIP84c7: dss-kiln Parameters**
```

File: MIP84/MIP84.md [hunk 13]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
-### MIP84c8: Modify the Daily Collateral Purchase Rate
+Any Dai available in the Pause Proxy, whether from POVEDC or Starting Collateral, is used to accumulate Lido Staked ETH, and later EtherDai, using a dss-kiln implementation sourcing liquidity from a liquid DEX. After the launch of Elixir, the dss-kiln target will be the Elixir balancer pool.
 
-This component enables a Subproposal Process to modify the Daily Collateral Purchase Rate as defined in MIP84c7.
+* `sell`: Dai
+* `buy`: stETH
+* `hop`: 20 minutes
+* `lot`: 7000
+
+### MIP84c8: Modify the Dss-kiln parameters
```
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
### MIP84c8: Modify the dss-kiln parameters
```

File: MIP84/MIP84.md [hunk 14]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
+
+This component enables a Subproposal Process to modify the Dss-kiln parameters as defined in MIP84c7.
```
Comment (blimpa) [inline MIP84/MIP84.md]:
```suggestion
This process component establishes the process to modify the Dss-kiln parameters as defined in MIP84c7.
```

Comment (blimpa) [reply MIP84/MIP84.md]: Maybe

File: MIP84/MIP84.md [hunk 15]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
 
-All additional income generated by the Maker Protocol is also used to fund the Protocoled Owned Vault, unless Dai has reached 75% collateralization by decentralized assets. To enable this with the POVE, this MIP also defines the concept of the Emulated Surplus Buffer Upper Limit.
+All additional income generated by the Maker Protocol is used to fund the POVE, until Maker Governance implements a new burn engine that also draws from the Surplus Buffer alongside the POVE according to the long-term tokenomics defined by the Endgame Plan.
+
+To enable this with the POVE, this MIP also defines the concept of the Emulated Surplus Buffer Upper Limit.
 
 The Emulated Surplus Buffer Upper Limit is a quantity of Dai above the current surplus buffer lower limit.
 
-Each month, any Dai accumulated in the Surplus Buffer above the Emulated Surplus Buffer Upper Limit is transferred to the Pause Proxy to be deployed in the POVE.
+Each month, any Dai accumulated in the Surplus Buffer above the Emulated Surplus Buffer Upper Limit is transferred to the Pause Proxy to be deployed in the POVE. This is calculated and executed AFTER first calculating and modifying the new POVEDC, if applicable.
```
Comment (blimpa) [inline MIP84/MIP84.md:168]:
```suggestion
Each month, any Dai accumulated in the Surplus Buffer above the Emulated Surplus Buffer Upper Limit is transferred to the Pause Proxy to be deployed in the POVE. This is calculated and executed _after_ first calculating and modifying the new POVEDC, if applicable.
```

File: MIP84/MIP84.md [hunk 16]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
+### MIP84c11: Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling
+
+To prevent overleverage of the POVE due to unsustainable price appreciation of the POVE's collateral assets, this MIP defines virtual price ceilings for POVEC. If the price of the collateral assets is above the POVEC virtual price ceilings, they are only counted as being equal to the virtual price ceiling when calculating the POVEDC.
+
+The POVEC Virtual Price Ceiling of ETH (For Lido Staked ETH or EtherDai): 1500 USD
+
+
+### MIP84c12: Modify Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling
+
+This component enables a Subproposal Process to modify the POVEC Virtual Price Ceiling defined in MIP84c11.
```
Comment (blimpa) [inline MIP84/MIP84.md:197]:
```suggestion
This process component establishes the process to modify the POVEC Virtual Price Ceiling defined in MIP84c11.
```

Comment (blimpa) [reply MIP84/MIP84.md:197]: Maybe

File: MIP84/MIP84.md [hunk 17]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
+- **Governance Cycle**: Monthly.
 
-To prevent overleverage of the POVE due to unsustainable price appreciation of the POVE's collateral assets, this MIP defines virtual price ceilings for POVEC. If the price of the collateral assets are above the POVEC virtual price ceilings, they are only counted as being equal to the virtual price ceilings when calculating the POVEDC.
+Facilitators with relevant mandates may propose weekly polls to amend the Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling when they deem necessary.
 
-The POVEC Virtual Price Ceiling of ETH (For Lido Staked ETH or EtherDai): 2000 USD
+MIP84c12 subproposals must use the template located at [MIP84c12-Subproposal-Template.md](https://github.com/makerdao/mips/blob/master/MIP84/MIP84c12-Subproposal-Template.md).
 
-The POVEC Virtual Price Ceiling of LDO: 3 USD
+# MIP84c13: Long-term replacement of the Protocol-Owned Vault Emulation
```
Comment (blimpa) [inline MIP84/MIP84.md:211]:
```suggestion
# MIP84c13: Long-term Replacement of the Protocol-Owned Vault Emulation
```

File: MIP84/MIP84.md [hunk 18]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
 
-### MIP84c11: Modify Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling
+The POVE is not meant to be a permanent feature of the Endgame, rather it is a short-term solution that allows Maker to quickly begin accumulating staked ETH, which enables it to be done at a slower rate, reducing risks.
```
Comment (blimpa) [inline MIP84/MIP84.md:213]:
```suggestion
The POVE is not meant to be a permanent feature of the Endgame; rather, it is a short-term solution that allows Maker to quickly begin accumulating staked ETH at a slower rate, reducing risks.
```

Comment (blimpa) [reply MIP84/MIP84.md:213]: Not 100% sure about this one

File: MIP84/MIP84.md [hunk 19]
```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
...
 
-This component enables a Subproposal Process to modify the POVEC Virtual Price Ceiling defined in MIP84c10.
+The POVE must eventually be replaced by a fully smart contract based, automated, Protocol-Owned Vault. The Endgame Plan Launch MIPs must contain an Endgame Product MIP for the Protocol-Owned Vault that will eventually replace the POVE.
```
Comment (blimpa) [inline MIP84/MIP84.md:215]:
```suggestion
The POVE must eventually be replaced by a fully smart contract-based, automated, protocol-owned Vault. The Endgame Plan Launch MIPs must contain an Endgame Product MIP for the protocol-owned vault that will eventually replace the POVE.
```

File: MIP84/MIP84c10-Subproposal-Template.md [hunk 20]
```diff
@@ -0,0 +1,27 @@
...
+
+## Paragraph Summary
+
+## Specification
+
+### Motivation
+
+- Why are you proposing this modification to the Emulated Surplus Buffer Upper Limit?
+
+### New Emulated Surplus Buffer Upper Limit parameter
```
Comment (blimpa) [inline MIP84/MIP84c10-Subproposal-Template.md:25]:
```suggestion
### New Emulated Surplus Buffer Upper Limit Parameter
```

File: MIP84/MIP84c12-Subproposal-Template.md [hunk 21]
```diff
@@ -0,0 +1,27 @@
...
+
+## Paragraph Summary
+
+## Specification
+
+### Motivation
+
+- Why are you proposing this modification to the POVEC Virtual Price Ceiling?
+
+### New POVEC Virtual Price Ceiling parameter
```
Comment (blimpa) [inline MIP84/MIP84c12-Subproposal-Template.md]:
```suggestion
### New POVEC Virtual Price Ceiling Parameter
```

File: MIP84/MIP84c6-Subproposal-Template.md [hunk 22]
```diff
@@ -0,0 +1,27 @@
...
+
+## Paragraph Summary
+
+## Specification
+
+### Motivation
+
+- Why are you proposing this modification to the state?
+
+### New collateralization ratio
```
Comment (blimpa) [inline MIP84/MIP84c6-Subproposal-Template.md]:
```suggestion
### New Collateralization Ratio
```

File: MIP84/MIP84c8-Subproposal-Template.md [hunk 23]
```diff
@@ -0,0 +1,34 @@
+# MIP84c8: Modify the Dss-kiln parameters
```
Comment (blimpa) [inline MIP84/MIP84c8-Subproposal-Template.md]:
```suggestion
# MIP84c8: Modify the dss-kiln Parameters
```

File: MIP84/MIP84c8-Subproposal-Template.md [hunk 24]
```diff
@@ -0,0 +1,34 @@
...
+
+## Paragraph Summary
+
+## Specification
+
+### Motivation
+
+- Why are you proposing this modification to the dss-kiln parameters?
+
+### New dss-kiln parameters
```
Comment (blimpa) [inline MIP84/MIP84c8-Subproposal-Template.md]:
```suggestion
### New dss-kiln Parameters
```

File: MIP84/MIP84c8-Subproposal-Template.md [hunk 25]
```diff
@@ -0,0 +1,34 @@
...
+
+- This section contains the new dss-kiln parameters that will override the current parameters in MIP84c7
+
+- The following parameters must be specified:
+
+   * `sell`: The token to be deposited into DssKiln and sold.
```
Comment (blimpa) [inline MIP84/MIP84c8-Subproposal-Template.md]:
```suggestion
  * `sell`: The token to be deposited into DssKiln and sold.
```

**Review by blimpa — CHANGES_REQUESTED** [review]
Mostly minutae

**Comment by ghost** [comment]
A style comment - PE use "DssKiln" ([source](https://forum.makerdao.com/t/dsskiln-a-permissionless-token-purchasing-tool/14943/12) )- in the text of this MIP both "dss-kiln" and "DssKiln" are used, can we make it consistent?

File: MIP84/MIP84.md (hunk 18 above)
Comment (ghost) [reply MIP84/MIP84.md:213]: He's saying it's quicker than a completely automated solution but it doesn't make a whole lot of sense as written. How about the below?

```suggestion
The POVE is not meant to be a permanent feature of the Endgame, rather it is a short-term solution that allows Maker to quickly begin accumulating staked ETH at a steady rate, reducing risks.
```
